### ISSUE TITLE
Doesn't compile with Java 11

### ISSUE THREAD

**Issue by lagenorhynque** [fpinscala/fpinscala#570]
```
exercises/src/main/scala/fpinscala/parsing/Parsers.scala:28:39: value drop is not a member of java.util.stream.Stream[String]
```

https://github.com/fpinscala/fpinscala/blob/master/exercises/src/main/scala/fpinscala/parsing/Parsers.scala#L28

[`lines`](https://docs.oracle.com/en/java/javase/12/docs/api/java.base/java/lang/String.html#lines()) method (which returns `java.util.stream.Stream`) was newly added to `java.lang.String` in Java 11, so you should use [`linesIterator`](https://www.scala-lang.org/api/current/scala/collection/immutable/StringOps.html#lines:Iterator[String]) method instead if you'd like to compile with Java 11 or above.

cf. https://github.com/scala/bug/issues/11125

**Comment by jlee9595** [comment]
I also experienced confusion with this but your fix resolved, thank you!

**Comment by Freyert** [comment]
Before:

`input.lines.drop(line-1).next`

After:

`input.linesIterator.drop(line-1).next`

Excellent!

**Comment by mpilquist** [comment]
Fixed as of 7c0cf46d9b9f8b41a27c1b46872cdaf76febd7d7
